test/users/test-delete.js

```diff
@@ -0,0 +1,17 @@
+const { assert } = require('chai');
+const { dispatch, mock_api, url_for, user } = require('../conftest');
+const handlers = require('../../app/users/handlers');
+
+
+describe('users', () => {
+  describe('delete', () => {
+    it('deletes user and redirects to list', () => {
+      mock_api().delete(`/users/${escape(user.auth0_id)}/`).reply(200, user);
```

Comment:
Sorry if I'm being a pain with these comments. May be a good idea to test that this call to the backend was made as well. But ignore me if it's too much hassle.

Reply:
Yep, I missed that. Thanks.